docs/kratos/manage-identities/40_invite-users.mdx

```diff
@@ -0,0 +1,105 @@
+---
+id: invite-users
+title: Learn how to invite users to Ory-powered applications
+sidebar_label: Invite users
+---
+
+# Invite users
+
+If your application must be accessible only to a hand-picked group of specific individuals, and you don't want to make
+registration publicly available, you can send invitations to selected users.
+
+The process of inviting users to an Ory-powered application follows these steps:
+
+1. You create a basic user account for the user you want to invite. You use their email address as the account identifier.
+2. You trigger account recovery for the user account you created.
+3. You get the recovery link from the API response and send it to the user's email address.
+4. The user performs account recovery which forces them to define new credentials.
+5. The user gets a fully functional account with the credentials they defined.
+
+:::info
+
+Automating this process requires writing custom code that relies on the specifics of your implementation.
+
+:::
+
+## Required API calls
```

Comment:
I was a bit confused by this subheading at first. Maybe consider changing it to something like "Procedure" or "Procedure to invite users"?

Reply:
Changed to "Procedure" 👍🏻